scalanativelib/src/ScalaNativeModule.scala

```diff
@@ -87,7 +87,14 @@ trait ScalaNativeModule extends ScalaModule { outer =>
 
   def logLevel: Target[NativeLogLevel] = T{ NativeLogLevel.Info }
 
-  private def releaseModeInput = T.input(sys.env.get("SCALANATIVE_MODE").map(ReleaseMode.valueOf))
+  private def releaseModeInput = T.input(
```

Comment:
I think making this protected is ok, e.g. to re-use it. Same for next change.

Reply:
Done ✅ 
Made all the inputs I added protected.